stevensblueprint/aad-admin
# Mentee / Mentor Matching Page

# Description

Completed Matching Page. It includes:
- Instructions
- Autocomplete input for selecting top 5
  - basic zod validation for selecting 5
- Drag and Drop for ordering top 5 by preference
- All 2 steps are inside of a Stepper so that the second cannot be completed without the first
- Mobile Friendly 
- Submit Notification


## Steps to test

While logged in, go to the `/matching` route to see it. Should be prompted to sign in if you are not. 


## Preview
Step 1
<img width="1022" alt="Screenshot 2024-04-10 at 3 26 34 AM" src="https://github.com/stevensblueprint/aad-admin/assets/43704084/5b2c9cd6-fc65-4abe-9374-cf4532393794">

Step 2 
<img width="1022" alt="Screenshot 2024-04-10 at 3 26 57 AM" src="https://github.com/stevensblueprint/aad-admin/assets/43704084/f71c2219-8efc-4aa8-95af-211e735ab5b5">

Relates to #90
